scripts/generate-app-config/generateExtensionConfig.js

```diff
@@ -0,0 +1,23 @@
+const ImportAggregator = require('./generation-objects/ImportAggregator');
+
+/**
+* Generates the file representing app name configuration.
+*/
+const generateExtensionConfig = (extensionConfig) => {
+  const imports = new ImportAggregator();
+
+
+  const config = extensionConfig.map(ext => (
+    {
+      icon: imports.addImport(ImportAggregator.relativePath(ext.iconPath)),
+      key: ext.key,
+      text: ext.text,
+      component: imports.addReactLazyImport(ImportAggregator.relativePath(ext.componentPath)),
```

Comment:
Won't extensions always be visible on the site page? Do we need to lazy load them?

Reply:
We're lazy loading the content displayed in the modal, the site search or issue page.